src/css/application.css

```diff
@@ -473,6 +473,20 @@ body {
   cursor: pointer;
 }
 
+.errorList-error-message {
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+
+  -ms-word-break: break-all;
+  word-break: break-all;
```

Comment:
I don’t think we want this property—won’t it cause words to break even if they’re not one long continuous word? Would rather have normal whitespace wrapping unless it is the specific case you’re targeting here (one very long unbroken “word”) which I think should be handled by `word-wrap`/`overflow-wrap`.